src/bundle/Resources/views/themes/admin/content/edit/edit.html.twig

```diff
@@ -7,11 +7,31 @@
 {% endblock %}
 
 {% block page_title %}
-    {% include '@ezdesign/content/page_title_edit.html.twig' with { 
+    {% embed '@ezdesign/content/page_title_edit.html.twig' with {
         action_name: 'editing'|trans|desc('Editing'),
         title: content.name,
+        content: content,
         description: content_type.description
     } %}
+        {% block close_button %}
+            {% if without_close_button is not defined or without_close_button != true %}
+                {% set referrer_location = is_published ? location : parent_location %}
+                <a class="ez-content-edit-container__close"
+                   href="{{ path('ezplatform.content.draft.edit.cancel', {
+                       'contentId': content.id,
+                       'referrerLocationId': referrer_location.id,
+                       'versionNo': content.versionInfo.versionNo,
+                       'languageCode': language.languageCode
+                   }) }}"
```

Comment:
Could you please extract 

```
path('ezplatform.content.draft.edit.cancel', {
     'contentId': content.id,
     'referrerLocationId': referrer_location.id,
     'versionNo': content.versionInfo.versionNo,
     'languageCode': language.languageCode
 })
```

value to variable ? This should improve readability